pkg/hubble/metrics/flows-to-world/handler_test.go

```diff
@@ -0,0 +1,199 @@
+				TCP: &flowpb.TCP{DestinationPort: 80},
+			},
+		},
+	})
+	// Drop reason is not "Policy denied".
+	h.ProcessFlow(context.Background(), &flowpb.Flow{
+		Verdict:        flowpb.Verdict_DROPPED,
+		EventType:      &flowpb.CiliumEventType{Type: monitorAPI.MessageTypeDrop},
+		DropReasonDesc: flowpb.DropReason_STALE_OR_UNROUTABLE_IP,
+		L4: &flowpb.Layer4{
+			Protocol: &flowpb.Layer4_TCP{
+				TCP: &flowpb.TCP{DestinationPort: 80},
+			},
+		},
+		Source: &flowpb.Endpoint{Namespace: "src-a"},
+		Destination: &flowpb.Endpoint{
+			Labels: []string{"reserved:world"},
+		},
+		DestinationNames: []string{"cilium.io"},
+	})
+	// Flow is a reply.
+	h.ProcessFlow(context.Background(), &flowpb.Flow{
+		Verdict:   flowpb.Verdict_FORWARDED,
+		EventType: &flowpb.CiliumEventType{Type: monitorAPI.MessageTypeTrace},
+		L4: &flowpb.Layer4{
+			Protocol: &flowpb.Layer4_TCP{
+				TCP: &flowpb.TCP{DestinationPort: 80},
+			},
+		},
+		Source: &flowpb.Endpoint{Namespace: "src-a"},
+		Destination: &flowpb.Endpoint{
+			Labels: []string{"reserved:world"},
+		},
+		DestinationNames: []string{"cilium.io"},
+		IsReply:          wrapperspb.Bool(true),
+	})
+	assert.NoError(t, testutil.CollectAndCompare(h.flowsToWorld, strings.NewReader("")))
+}
+
+func TestFlowsToWorldHandler_AnyDrop(t *testing.T) {
+	registry := prometheus.NewRegistry()
+	opts := api.Options{"sourceContext": "namespace", "destinationContext": "dns|ip", "any-drop": ""}
+	h := &flowsToWorldHandler{}
+	assert.NoError(t, h.Init(registry, opts))
+	assert.NoError(t, testutil.CollectAndCompare(h.flowsToWorld, strings.NewReader("")))
+	flow := flowpb.Flow{
+		Verdict:        flowpb.Verdict_DROPPED,
+		DropReasonDesc: flowpb.DropReason_STALE_OR_UNROUTABLE_IP,
+		EventType:      &flowpb.CiliumEventType{Type: monitorAPI.MessageTypeDrop},
+		L4: &flowpb.Layer4{
+			Protocol: &flowpb.Layer4_TCP{
+				TCP: &flowpb.TCP{DestinationPort: 80},
+			},
+		},
+		Source: &flowpb.Endpoint{Namespace: "src-a"},
+		Destination: &flowpb.Endpoint{
+			Labels: []string{"reserved:world"},
+		},
+		DestinationNames: []string{"cilium.io"},
+	}
+	h.ProcessFlow(context.Background(), &flow)
+	expected := strings.NewReader(`# HELP hubble_flows_to_world_total Total number of flows to reserved:world
+# TYPE hubble_flows_to_world_total counter
+hubble_flows_to_world_total{destination="cilium.io",protocol="TCP",source="src-a",verdict="DROPPED"} 1
+`)
+	assert.NoError(t, testutil.CollectAndCompare(h.flowsToWorld, expected))
+}
+
+func TestFlowsToWorldHandler_IncludePort(t *testing.T) {
+	registry := prometheus.NewRegistry()
+	opts := api.Options{"sourceContext": "namespace", "destinationContext": "dns|ip", "port": ""}
+	h := &flowsToWorldHandler{}
+	assert.NoError(t, h.Init(registry, opts))
+	assert.NoError(t, testutil.CollectAndCompare(h.flowsToWorld, strings.NewReader("")))
+	flow := flowpb.Flow{
+		Verdict:   flowpb.Verdict_FORWARDED,
+		EventType: &flowpb.CiliumEventType{Type: monitorAPI.MessageTypeDrop},
```

Comment:
tiny nit (doesn't affect correctness):
```suggestion
		EventType: &flowpb.CiliumEventType{Type: monitorAPI.MessageTypeTrace},
```